### PR TITLE
feat(test): add e2e test for timeout on config

### PR DESCRIPTION
**Pull request by barbmarcio**
This PR adds an E2E test to simulate timeout being part of the config

Needs to be merged after ([#11](https://github.com/Topsort/topsort.js/pull/11))

### CODE REVIEW

File: e2e/auctions.test.ts [hunk 1]
```diff
@@ -97,4 +98,51 @@ test.describe("Create Auction via Topsort SDK", () => {
...
+          {
+            type: "listings",
+            slots: 3,
+            category: { id: "cat123" },
+            geoTargeting: { location: "US" },
+          },
+        ],
+      };
+      if (typeof window.sdk.createAuction === "undefined") {
+        throw new Error("Global function `createAuction` is not available.");
```
Comment (jbergstroem) [inline e2e/auctions.test.ts:137]: can this happen? if not, strange logic to have in a test?

File: e2e/auctions.test.ts [hunk 2]
```diff
@@ -97,4 +98,51 @@ test.describe("Create Auction via Topsort SDK", () => {
...
+      await delay(2000);
+      await route.fulfill({ json: mockAPIResponse });
+    });
+
+    await page.goto(playwrightConstants.host);
+    const result = await page.evaluate(async () => {
+      const startTime = Date.now();
+      const config = {
+        apiKey: "rando-api-key",
+        timeOut: 2000,
```
Comment (jbergstroem) [inline e2e/auctions.test.ts:123]: i would test a response that takes too long; so set timeout to 50ms

**Comment by barbmarcio** [comment]
Can we land this?

**Review by jbergstroem — CHANGES_REQUESTED** [review]
I would still like to see at test that confirms that the request has timed out. Measuring how much time was spent is not really testing our logic for how to deal with the timeout.

**Review by jbergstroem — APPROVED** [review]

**Pull request merged**